jobs/consul_agent_windows/monit

```diff
@@ -0,0 +1,9 @@
+{
+  "processes": [
+    {
+      "name": "consul",
+      "executable": "C:\\var\\vcap\\packages\\confab-windows\\bin\\confab.exe",
+      "args": [ "start", "", "-config-file", "/var/vcap/jobs/consul_agent_windows/confab.json", "-foreground" ]
+    }
```

Comment:
The non-windows has a stop command (https://github.com/cloudfoundry-incubator/consul-release/blob/master/jobs/consul_agent/monit#L5) which does a best effort `consul leave`, what happens in the windows case?